### DISCUSSION TITLE
请问如何自定义一个没有证毕符号的命题环境？

### DISCUSSION THREAD

**Discussion by Juelianqvq** [stone-zeng/fduthesis#243] [Q&A] [answered]
![屏幕截图 2022-03-01 214153](https://user-images.githubusercontent.com/42963620/156179826-a2c4dbed-d4f1-4d23-904e-61a02df151dd.png)
如图，想要用newtheorem新建一个不带证毕符号的命题环境，有什么好办法吗？

**Comment by sikouhjw** [comment, marked as answer]
1. `fduthesis` 默认情况下，使用 `\newtheorem` 并不会产生证毕符号
2. 请提供你改动后的代码

测试代码
```tex
\documentclass{fduthesis}
\newtheorem{test}{命题}
\begin{document}
\mainmatter
\chapter{test}
\begin{test}
  test
\end{test}
\begin{test}
  test
\end{test}
\end{document}
```

**Reply by Juelianqvq** [reply]
\newtheorem*{prop1}{命题} 按这样写会出现证毕，把星号去除后证毕会消失。 但是全都这样改之后，命题的序号会出现新问题。
例如：代码是\newtheorem{prop2}{命题2.2}，命题名称会变成图中所示。
![屏幕截图 2022-03-01 220149](https://user-images.githubusercontent.com/42963620/156183086-43c70dea-24d4-46de-87c9-c821d0c38060.png)

**Reply by sikouhjw** [reply]
> \newtheorem*{prop1}{命题} 按这样写会出现证毕，把星号去除后证毕会消失。 但是全都这样改之后，命题的序号会出现新问题。 例如：代码是\newtheorem{prop2}{命题2.2}，命题名称会变成图中所示。 ![屏幕截图 2022-03-01 220149](https://user-images.githubusercontent.com/42963620/156183086-43c70dea-24d4-46de-87c9-c821d0c38060.png)

为什么要手动编号？自动编号有什么地方不够用吗？

**Reply by Juelianqvq** [reply]
可以请教下怎么进行自动编号吗？如果把2.2去除，所有命题编号全都变成了命题2.1，谢谢您耐心的解答！

**Reply by sikouhjw** [reply]
> 可以请教下怎么进行自动编号吗？如果把2.2去除，所有命题编号全都变成了命题2.1，谢谢您耐心的解答！

正常使用就是自动编号的。
```tex
\documentclass{fduthesis}
\newtheorem{test}{命题}
\begin{document}
\mainmatter
\chapter{test}
\begin{test}
  test
\end{test}
\begin{test}
  test
\end{test}
\end{document}
```
如果你编译这段代码还是不能自动编号，请检查你的模板是否为最新版（从 Code 下载）、发行版是否为 texlive 2021、有无报错。

**Reply by Juelianqvq** [reply]
十分感谢作者！问题已解决！问题出在我想要命题2.2的时候重新\newtheorem了，真的谢谢啦！
